controllers.js

```diff
@@ -0,0 +1,12 @@
+const {dataGetTopics} = require("./models")
+
+// TASK 3
+const getTopics = (req,res)=>{
+    return dataGetTopics(req,res).then(
```

Comment:
your model doesn't need a req, res, next argument structure. It is only the controller that deals with the request and response. If anything from the request is needed to be passed into the model, it should be stored to a variable and then that variable should be passed to the the model